client/app/services/event-notifications.service.js

```diff
@@ -42,21 +42,73 @@
       }
     };
 
+    function miqFormatNotification(text, bindings) {
+      var str = __(text);
+      lodash.each(bindings, function (value, key) {
+        str = str.replace(new RegExp('%{' + key + '}', 'g'), value.text);
+      });
+
+      return str;
+    }
+
     doReset();
 
+    var cable = ActionCable.createConsumer('/ws/notifications');
+
+    cable.subscriptions.create('NotificationChannel', {
+      disconnected: function () {
+        // eslint-disable-next-line consistent-this
```

Comment:
Agree with @himdel it is 💯% an angular thing, that being said, in the context of angular I am a **very** strong proponent of `vm = this` always forever :finnadie: 

Consistency throughout will be kindness to the future...